astropy/cosmology/units.py

```diff
@@ -107,6 +132,21 @@ def redshift_hubble(cosmology=None, **atzkw):
     -------
     `~astropy.units.equivalencies.Equivalency`
         Equivalency between redshift and Hubble parameter and little-h unit.
+
+    Examples
+    --------
+    >>> import astropy.units as u
+    >>> import astropy.cosmology.units as cu
+    >>> from astropy.cosmology import WMAP9
+
+    >>> z = 1100 * cu.redshift
+    >>> equivalency = cu.redshift_hubble(WMAP9)  # construct equivalency
+
+    >>> z.to(u.km / u.s / u.Mpc, equivalency)
```

Comment:
Here, also good to add `# doctest: +FLOAT_CMP`

Reply:
Done!